seed/views/v3/organizations.py

```diff
@@ -129,3 +339,999 @@ def column_mappings(self, request, pk=None):
+        """
+        org_id = pk
+
+        if org_id is None:
+            return JsonResponse({
+                'status': 'error',
+                'message': 'no organization_id sent'
+            }, status=status.HTTP_400_BAD_REQUEST)
+
+        try:
+            org = Organization.objects.get(pk=org_id)
+        except Organization.DoesNotExist:
+            return JsonResponse({
+                'status': 'error',
+                'message': 'organization does not exist'
+            }, status=status.HTTP_404_NOT_FOUND)
+        if (
+            not request.user.is_superuser and
+            not OrganizationUser.objects.filter(
+                user=request.user,
+                organization=org,
+                role_level__in=[ROLE_OWNER, ROLE_MEMBER, ROLE_VIEWER]
+            ).exists()
+        ):
+            # TODO: better permission and return 401 or 403
+            return JsonResponse({
+                'status': 'error',
+                'message': 'user is not the owner of the org'
+            }, status=status.HTTP_403_FORBIDDEN)
+
+        return JsonResponse({
+            'status': 'success',
+            'organization': _dict_org(request, [org])[0],
+        })
+
+    @swagger_auto_schema(
+        request_body=AutoSchemaHelper.schema_factory(
+            {
+                'organization_name': 'string',
+                'user_id': 'integer',
+            },
+            required=['organization_name', 'user_id'],
+            description='Properties:\n'
+                        '- organization_name: The new organization name\n'
+                        '- user_id: The user ID (primary key) to be used as the owner of the new organization'
+        )
+    )
+    @api_endpoint_class
+    @ajax_request_class
+    @has_perm_class('requires_parent_org_owner')
+    def create(self, request):
+        """
+        Creates a new organization.
+        """
+        body = request.data
+        user = User.objects.get(pk=body['user_id'])
+        org_name = body['organization_name']
+
+        if Organization.objects.filter(name=org_name).exists():
+            return JsonResponse({
+                'status': 'error',
+                'message': 'organization name already exists'
+            }, status=status.HTTP_409_CONFLICT)
+
+        org, _, _ = create_organization(user, org_name, org_name)
+        return JsonResponse(
+            {
+                'status': 'success',
+                'message': 'organization created',
+                'organization': _dict_org(request, [org])[0]
+            }
+        )
+
+    @api_endpoint_class
+    @ajax_request_class
+    @method_decorator(permission_required('seed.can_access_admin'))
+    @action(detail=True, methods=['DELETE'])
+    def inventory(self, request, pk=None):
+        """
+        Starts a background task to delete all properties & taxlots
+        in an org.
+        """
+        return JsonResponse(tasks.delete_organization_inventory(pk))
+
```

Comment:
When testing the return response was misleading.  
```
{
  "status": "not-started",
  "status_message": "",
  "progress": 0,
  "progress_key": ":1:SEED:delete_organization:PROG:4",
  "unique_id": "4",
  "func_name": "delete_organization",
  "message": null,
  "stacktrace": null,
  "summary": null,
  "total": null
}
```
The endpoint still successfully deletes an org but does it make sense to include a success json response here?

Reply:
I think in this case it's ok to not say "success" immediately because we've started a job, and it's the job's responsibility to say whether the delete was successfully or not